CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.17.0
```

Comment:
version 0.17.0 is already tagged and released, see #61 
you should update the changelog accordingly and go for 0.18.0